components/InspirationPost.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import Card from 'react-bootstrap/Card';
+import ModalComp from '../components/Modal';
+import CreateInspo from '../components/CreateInspo';
+import styled from 'styled-components';
+
+
+export default function InspirationPost({data, user}) {
+    return (
+        <>
+            <ModalComp title='Create Inspiration'>
+                <CreateInspo user={user} />
+            </ModalComp>
+            {
+            data.map((d) => (
+                <div key={d.id} className='text-white'>
+                    <ImageCardStyle>
+                        <Card className="imageCard">
+                            <Card.Img class="cardImage" src={d.image} alt="_1" height="100%" width="100%"/>
```

Comment:
should it be 'className' instead of 'class' maybe that's why the links are broken ?